install/systemd.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Run Airsonic with Systemd
+permalink: /docs/install/systemd/
+---
+##### Systemd
+
+Systemd is an init system for most linux systems. It allows one to run
+airsonic on boot.
```

Comment:
I'm going to be maniac here !

Please no new line if this isn't for a new paragraph.